pom.xml

```diff
@@ -67,9 +68,9 @@
         </dependency>
 
         <dependency>
-            <groupId>org.testcontainers</groupId>
-            <artifactId>kafka</artifactId>
-            <version>1.19.1</version>
+            <groupId>no.shhsoft</groupId>
+            <artifactId>k3a-embedded</artifactId>
+            <version>0.2.1+${kafka.version}</version>
```

Comment:
```suggestion
            <version>0.2.2+${kafka.version}</version>
```

Reply:
Thanks! Wonder if Renovate will handle the semver 2.0.0 plus sign (metadata) correctly.

Reply:
I think it should. I went through https://semver.org/ to check.

Reply:
Maybe it does not understand the `${kafka.version}` syntax, as it is a variable. The literal version `0.2.1+${kafka.version}` does not exist.